### ISSUE TITLE
Left over files on removal (feature request)

### ISSUE THREAD

**Issue by edp17** [sailfish-containers/harbour-containers#7]
When I uninstall the harbour-containers app, it leaves some files in the '/usr/share/harbour-containers/service/libs/__pycache__':
__init__.cpython-38.pyc
lxc.cpython-38.pyc
qxcompositor.cpython-38.pyc
It also leaves the configuration of the created container in the '/var/lib/lxc/<container-name>' folder and doesn't remove the files from the '/home/.lxc' folder.
I know these folders/files are kept because of SFOS' uninstall working order, but if you could include a button that runs a script, that is doing housekeeping before somebody decides to uninstall the app, that would be beneficial. (The downloaded data in 'home/.lxc' can be painful as eating a lot of storage.)

**Comment by Kabouik** [comment]
I agree about the `.pyc` file, but isn't it standard to keep configuration files and data files when uninstalling? `pkgbuild` even has a macro for configuration files to identify them and protect them from deletion at uninstall.

Default configuration file sure should be removed, but all configuration files here are tied to container names, and hence could be considered customized by user (mine *is* actually customized anyway because I added mount points).

Files in `/home/.lxc/` are data files in my opinion, and we wouldn't want that to be deleted with no prior notice at uninstall, people would expect to find them again if they reinstall the application. To me the `rootfs` in `/home/.lxc/` are pretty much like `.img` or `.vbox` files for VMs, they should persist even if the VM application is uninstalled.

However I agree that scripts/buttons to help cleaning before a full uninstall could be useful. I believe the `Settings > Destroy container` button is already doing that for `/home/.lxc/containername`, but it will work only for containers listed in the application, and hence would probably not be possible if the corresponding configuration file in `/var/lib/lxc/` has already been removed or moved. Actually, does the `Destroy` button delete the configuration file too? I didn't test that.
